src/core/directives/google-map-polyline-point.ts

```diff
@@ -0,0 +1,17 @@
+import {Directive, EventEmitter, Input, OnChanges, Output, SimpleChanges} from '@angular/core';
+
+@Directive({selector: 'sebm-google-map-polyline-point'})
+export class SebmGoogleMapPolylinePoint implements OnChanges {
```

Comment:
Please a description here (that describes that points are parts of the Polyline path)